benchmarks/bench_theory.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+import os
+import pathlib
+
+import pineko
+
+theory_obj = pineko.theory.TheoryBuilder(208, ["LHCB_Z_13TEV_DIMUON"])
+theory_obj_Hera = pineko.theory.TheoryBuilder(208, ["HERACOMBCCEM"])
```

Comment:
Respect naming convention, all lower case:
```suggestion
theory_obj_hera = pineko.theory.TheoryBuilder(208, ["HERACOMBCCEM"])
```

Reply:
And the rest accordingly